cmd/init.go

```diff
@@ -159,9 +160,21 @@ func getDevelopmentEnvironment(language string) (*model.Dev, string, error) {
 }
 
 func getDeploymentName(name string) string {
```

Comment:
It's better to split this into two functions. One for to generate the default name, and another to ask the user.